src/neuroglancer/chunk_manager/frontend.ts

```diff
@@ -224,9 +224,10 @@ export class ChunkQueueManager extends SharedObject {
         if (newState !== oldState) {
           switch (newState) {
             case ChunkState.GPU_MEMORY:
-              // console.log("Copying to GPU", chunk);
               chunk.copyToGPU(this.gl);
-              visibleChunksChanged = true;
+              if (chunk.constructor.name !== "ManifestChunk") {
+                visibleChunksChanged = true;
```

Comment:
This shouldn't be done by name, and also skipping the notification doesn't really fix the problem in general, because any other chunk update would also trigger the notification.  Additionally it will still redraw on the first fragment that is received so there will still be flicker if there is more than one fragment.  I think a different strategy is needed to avoid flicker.

Will some of the existing fragments be referenced by the new manifest, if they haven't changed, or does the fragment key depend on the root id?